app/views/home/index.html.haml

```diff
@@ -1,4 +1,3 @@
 %h1 Welcome
 
-%form{:action => "/users/sign_up"}
-  %input{:type => "submit", :value => "Sign up"}
+%a{:href => new_user_registration_path } Sign up
```

Comment:
Glad you made the change, but I would suggest you use the `link_to` helper. ;-)

Reply:
Good point! Will do!

Reply:
And please don't use hash rockets. No matter what the SO posts or the html2haml translator tell you.

Reply:
Got it!